README.md

```diff
@@ -18,7 +18,7 @@ Or install it yourself as:
 
     $ gem install plaid
 
-The gem supports Ruby 2.1+ only.
+The gem supports Ruby 2.2+ only.
```

Comment:
Any specific 2.2+ features used here?

Reply:
It's required by a dependency:

```rdoc-6.0.4 requires ruby version >= 2.2.2, which is incompatible with the current version, ruby 2.1.10p492```

Is it OK to upgrade to 2.2?

Reply:
ah I see. Then it's ok.